test/infra/aws/main.tf

```diff
@@ -0,0 +1,89 @@
+terraform {
+  required_version = ">= 0.12"
+}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+variable "aws_availability_zones" {
+  type        = list
+  description = "Availability zones to be used"
+  default     = ["us-west-2c"]
+}
+
+variable "vpc_cidr" {
+  description = "The CIDR used for vpc"
+  default     = "10.0.0.0/16"
+}
+
+variable "tags" {
+  description = "Map of tags to add to all resources"
+  type        = map
+  default     = {}
+}
+
+resource "aws_vpc" "konvoy_vpc" {
```

Comment:
Maybe I misunderstood but I thought we weren't going to create infra for every test run?
Instead create a SecurityGroup(if its already there it would be a noop) in the default VPC and Subnet with an S3 as the TF storage.

Reply:
No, you're absolutely right that is what we talked about. 

I didn't want to leave the security-group id or subnet-id in a public repo so i had to go with the templating appraoch. I'm not sure how to pull values out using an s3 backend so i did the local directory approach.

I'll try to change it more once I confirm the general idea works.

Reply:
Oh yeah good point about the IDs.

Instead of having this run by CI every time we could set it the IDs in CI and have another process run this when actually needed, its unlikely that the SG would be deleted by something or someone so even having docs how to run this locally to recreate the SG in the default VPC and what TC jobs need to be updated with the ID would be enough.

Reply:
Actually looking at the job logs it takes about a minute to create these resources, maybe we should just create it for each run.
Only thing is we may want to add a TC step at the end that runs `$(MAKE) devkit.run WHAT="make infra.aws.destroy"` to not leak resources.

Reply:
Added to the centos7 one and will add as more tests begin to use this environment